### PR TITLE
825 add region

### PR DESCRIPTION
**Pull request by dennis-kiewiet-myparcel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2298,6 +2329,12 @@ <h3 id="7_B_1">7.B.1 Overview</h3>
                         The address country code.
                     </p>
                     <p>
+                        <strong>region</strong><br>
+                        Data type: <a href="#region">region</a><br>
```
Comment (reindert-vetter) [inline index.html]: Moet worden:
`Data type: <a href="#string">region</a><br>`

**Review by reindert-vetter — CHANGES_REQUESTED** [review]
Ziet er goed uit!
Nog 1 puntje

File: index.html [hunk 2]
```diff
@@ -1616,9 +1629,12 @@ <h3 id="6_E_4">6.E.4 Example</h3>
   "data": {
     "search_results": {
       "shipments": [{
+        "reference_identifier" : "FOO-222-BAR-42",
         "recipient": {
           "cc": "NL",
+          "region": "Zuid-Holland",
           "city": "Hoofddorp",
+          "region": "Zuid-Holland",
```
Comment (reindert-vetter) [inline index.html]: dubbel

**Comment by janssenr** [comment]
Moet worden:

Data type: <a href="#string">string</a><br>

 

Van: Reindert [mailto:notifications@github.com] 
Verzonden: donderdag 18 mei 2017 11:39
Aan: myparcelnl/api <api@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: Re: [myparcelnl/api] 825 add region (#32)

 

@reindert-vetter requested changes on this pull request.

Ziet er goed uit!
1 puntje

  _____  

In index.html <https://github.com/myparcelnl/api/pull/32#discussion_r117202358> :

> @@ -2298,6 +2329,12 @@ <h3 id="7_B_1">7.B.1 Overview</h3>
                         The address country code.
                     </p>
                     <p>
+                        <strong>region</strong><br>
+                        Data type: <a href="#region">region</a><br>

Moet worden:
Data type: <a href="#string">region</a><br>

File: index.html (hunk 1 above)
Comment (janssenr) [reply index.html]: Moet worden:
Data type: <a href="#string">string</a><br>

**Review by reindert-vetter — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (dennis-kiewiet-myparcel) [reply index.html]: fixed

**Pull request merged**